localization/english/wc_laws_l_english.yml

```
@@ -0,0 +1,27 @@
+﻿l_english:
+ dark_authority:0 "[dark_authority|E]"
+ dark_authority_0:0 "Disparate Cults"
+ dark_authority_1:0 "Weak Covens"
+ dark_authority_2:0 "Black Sabbath"
+ dark_authority_3:0 "Unholy Katavasia"
```

Only russian slang uses this type of hymn as an "chaos" synonym

```suggestion
 dark_authority_3:0 "Unholy Congregations"
```